### PR TITLE
Feature lwdev 5296 mt wamp migration

### PR DESCRIPTION
**Pull request by mihail-yartsev**

### CODE REVIEW

File: src/Lykke.Frontend.WampHost.Core/Lykke.Frontend.WampHost.Core.csproj [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
   </PropertyGroup>
-
   <ItemGroup>
     <PackageReference Include="Lykke.Common" Version="4.3.0" />
     <PackageReference Include="Lykke.Job.CandlesProducer.Contract" Version="2.2.0" />
     <PackageReference Include="Lykke.Job.QuotesProducer.Contract" Version="1.0.0" />
     <PackageReference Include="Lykke.Job.TradesConverter.Contract" Version="1.0.7" />
     <PackageReference Include="Lykke.Logs" Version="3.5.1" />
     <PackageReference Include="Lykke.RabbitMqBroker" Version="6.0.0" />
+    <PackageReference Include="Lykke.MarginTrading.Contracts" Version="1.0.0-beta01" />
```
Comment (andrey-anokhin) [inline src/Lykke.Frontend.WampHost.Core/Lykke.Frontend.WampHost.Core.csproj]: Beta should not be used in test branch

File: src/Lykke.Frontend.WampHost/Lykke.Frontend.WampHost.csproj [hunk 2]
```diff
@@ -38,6 +38,7 @@
     <PackageReference Include="Lykke.Common" Version="4.3.0" />
     <PackageReference Include="Lykke.Common.ApiLibrary" Version="1.3.1" />
     <PackageReference Include="Lykke.Logs" Version="3.5.1" />
+    <PackageReference Include="Lykke.MarginTrading.Contracts" Version="1.0.0-beta02" />
```
Comment (andrey-anokhin) [inline src/Lykke.Frontend.WampHost/Lykke.Frontend.WampHost.csproj]: Beta

Comment (mihail-yartsev) [reply src/Lykke.Frontend.WampHost/Lykke.Frontend.WampHost.csproj]: fixed

File: src/Lykke.Frontend.WampHost.Core/Lykke.Frontend.WampHost.Core.csproj (hunk 1 above)
Comment (mihail-yartsev) [reply src/Lykke.Frontend.WampHost.Core/Lykke.Frontend.WampHost.Core.csproj]: fixed

**Pull request merged**